### PR TITLE
add custom audit recipe

### PR DESCRIPTION
**Pull request by brendankenny**
adds a custom gatherer/audit to our recipes. Happy to bikeshed on all of this :)

No rush on landing this, but we'll be using a version of this at I/O, so get in comments if you want to influence what we show there :)

### CODE REVIEW

File: docs/recipes/custom-audit/custom-config.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  // Run our tests along with all the default Lighthouse tests.
```
Comment (ebidel) [inline docs/recipes/custom-audit/custom-config.js]: "Run your custom tests...."

File: docs/recipes/custom-audit/custom-config.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      'searchable-gatherer'
+    ],
+  }],
+
+  // Add custom audit to the list of audits 'lighthouse:default' will run.
+  audits: [
+    'searchable-audit'
+  ],
+
+  // Create a new 'My site metrics' section in the default report for our results.
```
Comment (ebidel) [inline docs/recipes/custom-audit/custom-config.js]: How about a 1., 2., 3., 4. on these.

File: docs/recipes/custom-audit/readme.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Basic custom audit recipe for Lighthouse
+
+A hypothetical site measures the time from navigation start to when the page has initialized and the main search box is ready to be used. It saves that value in a global variable, `window.myLoadMetrics.searchableTime`.
+
+This Lighthouse [gatherer](searchable-gatherer.js)/[audit](searchable-audit.js) pair will take that value from the context of the page and test whether or not it stays below a test threshold.
+
+The config file tells Lighthouse where to find the gatherer and audit files, when to run them, and how to incoporate their output into the Lighthouse report.
```
Comment (ebidel) [inline docs/recipes/custom-audit/readme.md]: incorporate

File: docs/recipes/custom-audit/custom-config.js (hunk 1 above)
Comment (brendankenny) [inline docs/recipes/custom-audit/custom-config.js]: done

File: docs/recipes/custom-audit/custom-config.js (hunk 2 above)
Comment (brendankenny) [inline docs/recipes/custom-audit/custom-config.js]: done

File: docs/recipes/custom-audit/readme.md (hunk 3 above)
Comment (brendankenny) [inline docs/recipes/custom-audit/readme.md]: done

**Comment by benschwarz** [comment]
I didn't get a chance to review, but I was coming here to give it a thumbs up. 🌮

**Pull request merged**